src/React.AspNet/HtmlHelperExtensions.cs

```diff
@@ -63,10 +63,11 @@ private static IReactEnvironment Environment
 		/// <typeparam name="T">Type of the props</typeparam>
 		/// <param name="htmlHelper">HTML helper</param>
 		/// <param name="componentName">Name of the component</param>
-		/// <param name="props">Props to initialise the component with</param>
```

Comment:
"initialise" is correct in British English :) If you want to Americanise (Americanize?) the spelling, please do it in a separate pull request :)